1-exercises/D-do-while/exercise.js

```diff
@@ -8,7 +8,18 @@
 
 function evenNumbersSum(n) {
     // TODO
-}
+    const ourArray = []; 
```

Comment:
Hey Danti, I can see the approach your going for here, but it looks like you forgot to include some code - this file doesn't run and references a missing function `sumEvens`